lib/tasks/fake_data.rake

```diff
@@ -62,6 +80,13 @@ class FakeDataGenerator
         editor: user
     end
 
+    # User-saved stories
+    (@stories_count / 10).times do
+      user = users[Random.rand(@users_count-1)]
+      story = Story.all.sample
```

Comment:
Why not stick this in a `@stories` like `@categories`? I don't have a strong feeling on trading off a little memory vs speed in a one-off script like this, I'm just distracted by the inconsistency.

Reply:
Honestly, I was just trying to interact more with the database to learn that better. No good reason to do that, I'll shove them into arrays.